lib/flame_audio.dart

```diff
@@ -31,13 +33,26 @@ class FlameAudio {
     return audioCache.loop(file, volume: volume);
   }
 
+  void _warnWebFecthing() {
+    print(
+        'Prefetching audio is not supported by Audiplayers on web yet, to prefetch audio on web, try using the http package to get the file and the browser will handle cache');
```

Comment:
trailing comma ;)

Reply:
Full stop would be nicer than the trailing comma. :)

Reply:
Done